### PR TITLE
Update cmd options

### PR DESCRIPTION
**Pull request by rayzhou2017**

### CODE REVIEW

File: cmd/create/config.go [hunk 1]
```diff
@@ -20,9 +20,9 @@ func NewCmdCreateCfg() *cobra.Command {
 		},
 	}
 
-	clusterCfgCmd.Flags().StringVarP(&addons, "add", "", "", "Add plugins")
-	clusterCfgCmd.Flags().StringVarP(&name, "name", "", "", "Name of config file")
-	clusterCfgCmd.Flags().StringVarP(&clusterCfgPath, "config", "f", "", "cluster info config")
+	clusterCfgCmd.Flags().StringVarP(&addons, "add", "", "", "add plugins")
+	clusterCfgCmd.Flags().StringVarP(&name, "name", "", "", "cluster name")
+	clusterCfgCmd.Flags().StringVarP(&clusterCfgPath, "config", "f", "", "configuration file name")
```
Comment (rayzhou2017) [inline cmd/create/config.go:25]: what does the option look like? --config or -f?

File: cmd/create/config.go [hunk 2]
```diff
@@ -20,9 +20,9 @@ func NewCmdCreateCfg() *cobra.Command {
 		},
 	}
 
-	clusterCfgCmd.Flags().StringVarP(&addons, "add", "", "", "Add plugins")
-	clusterCfgCmd.Flags().StringVarP(&name, "name", "", "", "Name of config file")
-	clusterCfgCmd.Flags().StringVarP(&clusterCfgPath, "config", "f", "", "cluster info config")
+	clusterCfgCmd.Flags().StringVarP(&addons, "add", "", "", "add plugins")
+	clusterCfgCmd.Flags().StringVarP(&name, "name", "", "", "cluster name")
```
Comment (rayzhou2017) [inline cmd/create/config.go:24]: is this --name? Shall we support -a?

Comment (pixiake) [reply cmd/create/config.go:24]: `-n` can be supported. But `-n` may have other meanings，such as `namespace`.  So I didn't add it.

File: cmd/create/config.go (hunk 1 above)
Comment (pixiake) [reply cmd/create/config.go:25]: It should be modified here. The value of this parameter is the file path，So i think  `--config` should be deprecated 
or replaced with `--file` here.
I'll modify here later.

**Pull request merged**